encryptor/network/message.py

```diff
@@ -223,12 +230,22 @@ def read(self, content_type: Optional[ContentType] = None) -> Message:
 
         message_type = message.headers.content_type
 
+        if message_type == ContentType.FILE:
+            print(
+                f"New message from {self.endpoint_addr}: {message.headers.filename}\nSaving..."
+            )
+            message.write_to_file(keys_dir)
+        #            files_dir: Path = keys_dir / "files"
+        #            files_dir.mkdir(exist_ok=True)
+        #            file_path: Path = files_dir / message.headers.filename
+        #            file_path.write_bytes(message.content)
+
```

Comment:
This should be somewhere else, probably in the main GUI thread.

Reply:
done